app/src/main/java/com/zulip/android/models/Message.java

```diff
@@ -689,10 +689,13 @@ public void updateFromHistory(@NonNull MessageHistory history) {
         editDate = history.date;
     }
 
-    public boolean isHasBeenEdited() {
```

Comment:
I changed it since [hasBeenEdited is of type Boolean](https://github.com/zulip/zulip-android/blob/master/app/src/main/java/com/zulip/android/models/Message.java#L120) so on calling this method when hasBeenEdited is null, we would get a null pointer exception `Attempt to invoke virtual method 'boolean java.lang.Boolean.booleanValue()' on a null object reference`